v1.2/create-sequence.md

```diff
@@ -4,13 +4,15 @@ summary:
 toc: false
 ---
 
-<span class="version-tag">New in v1.2:</span> The `CREATE SEQUENCE` [statement](sql-statements.html)...
+<span class="version-tag">New in v2.0:</span> The `CREATE SEQUENCE` [statement](sql-statements.html) creates a new sequence in a database. Use a sequence to auto-increment integers in a table.
+
+{{site.data.alerts.callout_info}}Using sequences may slow down your application. Use the serial data type unless a sequence is preferred or required.{{site.data.alerts.end}}
```

Comment:
another important gotcha that should be called out: sequences can leave gaps. Users might expect that when they use a sequence they'll never have gaps (i.e. they'll see `1, 2, 3, …` for that column in their table) but there actually can be gaps if you roll back a transaction that uses a sequence. E.g.:

```
INSERT INTO my_sequence_using_table VALUES ('foo', 'bar');
=> gets id 1
BEGIN;
INSERT INTO my_sequence_using_table VALUES ('bloop', 'baz');
=> gets id 2
ROLLBACK;
-- at this point even though the row isn't inserted, id 2 is still considered taken.
-- the `ROLLBACK` doesn't "put it back" in the sequence
INSERT INTO my_sequence_using_table VALUES ('beep', 'blorp');
=> gets id 3
-- now we have a gap: 1 => 3
```

This is done for performance reasons, and matches PG. PG docs here: https://www.postgresql.org/docs/10/static/functions-sequence.html, specifically "To avoid blocking concurrent transactions that obtain numbers from the same sequence, a nextval operation is never rolled back; that is, once a value has been fetched it is considered used and will not be returned again. [...]"

Reply:
added. @jseldess - i've included two notes here back to back. kosher?